operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainStatus.java

```diff
@@ -71,10 +73,10 @@ public class DomainStatus {
           + "You cannot configure a limit for other types of failures, such as a Domain resource reference "
           + "to an unknown secret name; in which case, the retries are unlimited.")
   @Range(minimum = 0)
-  private Integer introspectJobFailureCount = 0;
+  private Integer introspectJobFailureCount = 0;  //todo remove this field
```

Comment:
I'm okay that this is a todo for a later PR, but just making sure that this is your intention.

Reply:
Yes, as noted in the description, I will be removing those fields in a subsequent PR.

Reply:
Since this is the second PR on which you've made such a comment, should I assume that putting TODOs in the code for a future PR is not a practice you endorse? I should keep that information in a separate place?

Reply:
It's not that -- todo's are okay... I needed more in the comment following the todo so that I don't have to look elsewhere for a statement of what needs to be done (at least simply) and if this is near-term work or for someday.